sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_path_client_async.py

```diff
@@ -622,8 +675,31 @@ async def _set_access_control_internal(self, options, progress_hook, max_batches
             error.continuation_token = last_continuation_token
             raise error
 
-    async def _rename_path(self, rename_source, **kwargs):
-        # type: (str, **Any) -> Dict[str, Any]
+    def _parse_rename_path(self, new_name: str) -> Tuple[str, str, Optional[str]]:
```

Comment:
This looks like something that could be a helper?

Reply:
Agreed, will take this feedback.